config/projects/google-fluentd.rb

```diff
@@ -8,13 +8,13 @@
 description "Google Fluentd: A data collector for Google Cloud Logging"
 
 install_dir     "/opt/google-fluentd"
-build_version   "1.5.9"
+build_version   "1.5.9.grpc.alpha.2"
 build_iteration 1
 
 # creates required build directories
 dependency "preparation"
 
-override :ruby, :version => '2.1.10'
+override :ruby, :version => '2.1.8'
```

Comment:
What about using Ruby 2.1.9 if the problem with the protobuf support is only a regex issue?